astropy/time/time_helper/function_helpers.py

```diff
@@ -0,0 +1,49 @@
+"""
+Helpers for overriding numpy functions in
+`~astropy.time.Time.__array_function__`.
+"""
+import numpy as np
+
+from astropy.units.quantity_helper.function_helpers import FunctionAssigner
+
+# TODO: Fill this in with functions that don't make sense for times
+UNSUPPORTED_FUNCTIONS = {}
+# Functions that take two time arguments and can be applied to their floating
+# point values
+TWOARG_HELPERS = {}
+
+function_helper = FunctionAssigner(TWOARG_HELPERS)
+
+
+@function_helper(helps={
+    np.linspace, np.greater, np.greater_equal, np.less, np.less_equal,
```

Comment:
`np.greater` etc are ufuncs, so would go via `__array_ufunc__` instead of `__array_function__`.  Maybe best kept for a separate PR (there was some work towards it already... can try to find...)